src/components/FilterTab/FilterTab.tsx

```diff
@@ -0,0 +1,111 @@
+import './FilterTab.css'
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect, useState } from 'react';
+import GeneNameInput from '../FilterInputs/GeneNameInput/GeneNameInput';
+import OrganismInput from '../FilterInputs/OrganismInput/OrganismInput';
+import SequenceLengthInput from '../FilterInputs/SequenceLengthInput/SequenceLengthInput';
+import AnnotationInput from '../FilterInputs/AnnotationInput/AnnotationInput'
+import ProteinInput from '../FilterInputs/ProteinInput/ProteinInput';
+import { fetchItems, removeItems } from '../../store/listItems';
+import { setNewValueFilter } from '../../store/filterOptions';
+import { setNewValueSort } from '../../store/sortOptions';
+import { setNewSelectedProteinName } from '../../store/selectedProtein';
+import { AnyAction } from 'redux';
+import { ThunkDispatch } from 'redux-thunk';
+
+
+const FilterTab = () => {
+    const dispatch: ThunkDispatch<any, any, AnyAction> = useDispatch();
+
+    //filters stored in redux;
+    const filterOptions = useSelector((state: any) => state.filterOptions);
+    //search text stored in redux;
+    const searchText = useSelector((state: any) => state.searchText);
+    
+    //determines when 'apply filters' button should be valid;
+    const [applyButtonValid, setApplyButtonValid] = useState(false);
+
+    /*
+        removes previous list items from redux;
+        sets filters to no filters (default);
+        sets sort to no sort (default);
+        fetches new list items with just searchText;
+    */
+    const cancelFunction = () => {
+        dispatch(removeItems());
+        dispatch(setNewValueFilter({
+            isOpen: false,
+            geneName: null,
+            organism: null,
+            sequenceLength__from: null,
+            sequenceLength__to: null, 
```

Comment:
```suggestion
            sequenceLengthFrom: null,
            sequenceLengthTo: null, 
```
two underscores - it is very unusual in JS variables naming